models/examples/e_05_contract.py

```diff
@@ -0,0 +1,39 @@
+from credmark.cmf.model import Model, EmptyInput
+from credmark.cmf.types import Contract
+from models.dtos.example import ExampleModelOutput
+
+from models.tmp_abi_lookup import CMK_ADDRESS
+
+
+@Model.describe(
+    slug='example.contract',
+    version='1.0',
+    display_name='Contract Usage Examples',
+    description='This model gives examples of the functionality available on the Contract class',
+    developer='Credmark',
+    input=EmptyInput,
+    output=ExampleModelOutput)
+class ExampleAddress(Model):
+    def run(self, _) -> ExampleModelOutput:
+        output = ExampleModelOutput(
+            github_url="https://github.com/credmark/credmark-models-py/blob/main/models/examples/e_05_contract.py",
+            documentation_url="https://developer-docs.credmark.com/en/latest/reference/credmark.cmf.types.contract.Contract.html")
+
+        output.log("This model demonstrates the functionality of the Contract class")
+
+        output.log("Contract is a subclass of Account, and is initialized with an address.")
+        contract = Contract(address=CMK_ADDRESS)
```

Comment:
Can we use a contract that's NOT a token, to introduce these concepts one at a time? We could use a CMK Vesting contract or an aave reserve or something

Reply:
Sure.